arangod/GeneralServer/AuthenticationFeature.cpp

```diff
@@ -171,12 +171,6 @@ void AuthenticationFeature::validateOptions(
     }
   }
 
-  if (_sessionTimeout <= 1.0) {
```

Comment:
Nitpicking: If we want to preserve the original behaviour, we would have to specify `minInclusive = false`.

Reply:
Fixed